xonsh/built_ins.py

```diff
@@ -720,6 +724,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_execer__ = execer
     builtins.__xonsh_commands_cache__ = CommandsCache()
     builtins.__xonsh_all_jobs__ = {}
+    builtins.__xonsh_enabled_builtin_commands__ = set(all_builtin_commands.keys())
```

Comment:
The removal would just be:
```
$ enable -n cat
```

No need to know where it lives.

Or maybe you're saying it could be confusing that `enable` wouldn't work on aliases?  To me, that's a reason to make sure `aliases` only contains user-specified aliases.